JWBestPracticeApps/Podfile

```diff
@@ -79,3 +83,16 @@ target 'Picture in Picture' do
   project 'Picture in Picture/Picture in Picture'
   common_JWPlayer
 end
+
+####################################
+#### JWPlayerTVKit tvOS targets ####
+####################################
+
+def common_JWPlayerTVKit
+    pod 'JWPlayerTVKit', '~> 1'
```

Comment:
It seems we need to specify the platform for tvOS as well. We can let the `pod install` script decide it automatically by removing the first line but it throws a few warnings for this.